packages/web-components/src/components/image/__stories__/image.stories.ts

```diff
@@ -24,6 +24,7 @@ import textNullable from '../../../../.storybook/knob-text-nullable';
50LXNpemU9IjEyIiBsZXR0ZXItc3BhY2luZz0iMGVtIj48dHNwYW4geD0iODc3IiB5PSIxNjEuNSI+U2VydmVybGVzczwvdHNwYW4+PC90ZXh0Pgo8cGF0aCBkPSJNMTAwMC41IDIzNC41SDg2Ni41VjI3NS41SDEwMDAuNVYyMzQuNVoiIHN0cm9rZT0iI0E4QThBOCIvPgo8dGV4dCBmaWxsPSIjMTYxNjE2IiB4bWw6c3BhY2U9InByZXNlcnZlIiBzdHlsZT0id2hpdGUtc3BhY2U6IHByZSIgZm9udC1mYW1pbHk9IklCTSBQbGV4IFNhbnMiIGZvbnQtc2l6ZT0iMTIiIGxldHRlci1zcGFjaW5nPSIwZW0iPjx0c3BhbiB4PSI4NzciIHk9IjI2MC41Ij5DRE48L3RzcGFuPjwvdGV4dD4KPHBhdGggZD0iTTEwMDAuNSAyODUuNUg4NjYuNVYzMjYuNUgxMDAwLjVWMjg1LjVaIiBzdHJva2U9IiNBOEE4QTgiLz4KPHRleHQgZmlsbD0iIzE2MTYxNiIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSIgc3R5bGU9IndoaXRlLXNwYWNlOiBwcmUiIGZvbnQtZmFtaWx5PSJJQk0gUGxleCBTYW5zIiBmb250LXNpemU9IjEyIiBsZXR0ZXItc3BhY2luZz0iMGVtIj48dHNwYW4geD0iODc3IiB5PSIzMDMuNSI+R292ZXJuYW5jZSwgcmlzazwvdHNwYW4+PC90ZXh0Pgo8dGV4dCBmaWxsPSIjMTYxNjE2IiB4bWw6c3BhY2U9InByZXNlcnZlIiBzdHlsZT0id2hpdGUtc3BhY2U6IHByZSIgZm9udC1mYW1pbHk9IklCTSBQbGV4IFNhbnMiIGZvbnQtc2l6ZT0iMTIiIGxldHRlci1zcGFjaW5nPSIwZW0iPjx0c3BhbiB4PSI4NzciIHk9IjMxOC41Ij4mIzM4OyBjb21wbGlhbmNlPC90c3Bhbj48L3RleHQ+CjxwYXRoIGQ9Ik0xMDAwLjUgNDM2LjVIODY2LjVWNDc3LjVIMTAwMC41VjQzNi41WiIgc3Ryb2tlPSIjQThBOEE4Ii8+Cjx0ZXh0IGZpbGw9IiMxNjE2MTYiIHhtbDpzcGFjZT0icHJlc2VydmUiIHN0eWxlPSJ3aGl0ZS1zcGFjZTogcHJlIiBmb250LWZhbWlseT0iSUJNIFBsZXggU2FucyIgZm9udC1zaXplPSIxMiIgbGV0dGVyLXNwYWNpbmc9IjBlbSI+PHRzcGFuIHg9Ijg3NyIgeT0iNDU0LjUiPkF1dG9tYXRlZDwvdHNwYW4+PC90ZXh0Pgo8dGV4dCBmaWxsPSIjMTYxNjE2IiB4bWw6c3BhY2U9InByZXNlcnZlIiBzdHlsZT0id2hpdGUtc3BhY2U6IHByZSIgZm9udC1mYW1pbHk9IklCTSBQbGV4IFNhbnMiIGZvbnQtc2l6ZT0iMTIiIGxldHRlci1zcGFjaW5nPSIwZW0iPjx0c3BhbiB4PSI4NzciIHk9IjQ2OS41Ij5kZXBsb3ltZW50PC90c3Bhbj48L3RleHQ+CjxwYXRoIGQ9Ik0xMTQzLjUgMjM0LjVIMTAwOS41VjI3NS41SDExNDMuNVYyMzQuNVoiIHN0cm9rZT0iI0E4QThBOCIvPgo8dGV4dCBmaWxsPSIjMTYxNjE2IiB4bWw6c3BhY2U9InByZXNlcnZlIiBzdHlsZT0id2hpdGUtc3BhY2U6IHByZSIgZm9udC1mYW1pbHk9IklCTSBQbGV4IFNhbnMiIGZvbnQtc2l6ZT0iMTIiIGxldHRlci1zcGFjaW5nPSIwZW0iPjx0c3BhbiB4PSIxMDIwIiB5PSIyNjAuNSI+RE5TPC90c3Bhbj48L3RleHQ+CjxwYXRoIGQ9Ik0xMTQzLjUgNDM2LjVIMTAwOS41VjQ3Ny41SDExNDMuNVY0MzYuNVoiIHN0cm9rZT0iI0E4QThBOCIvPgo8dGV4dCBmaWxsPSIjMTYxNjE2IiB4bWw6c3BhY2U9InByZXNlcnZlIiBzdHlsZT0id2hpdGUtc3BhY2U6IHByZSIgZm9udC1mYW1pbHk9IklCTSBQbGV4IFNhbnMiIGZvbnQtc2l6ZT0iMTIiIGxldHRlci1zcGFjaW5nPSIwZW0iPjx0c3BhbiB4PSIxMDIwIiB5PSI0NjEuNSI+TWFuYWdlbWVudDwvdHNwYW4+PC90ZXh0Pgo8cGF0aCBkPSJNMTY2LjUgNTYwLjVIMTUxLjVWNTc1LjVIMTY2LjVWNTYwLjVaIiBmaWxsPSIjRURGNUZGIiBzdHJva2U9IiMwRTYyRkQiLz4KPHBhdGggZD0iTTAuNSA1NDQuNUgxMTQ0LjUiIHN0cm9rZT0iI0M2QzZDNiIgc3Ryb2tlLWxpbmVjYXA9InNxdWFyZSIvPgo8cGF0aCBkPSJNMTM2LjUgNTI4LjVWMC41IiBzdHJva2U9IiNDNkM2QzYiIHN0cm9rZS1saW5lY2FwPSJzcXVhcmUiLz4KPHRleHQgZmlsbD0iIzE2MTYxNiIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSIgc3R5bGU9IndoaXRlLXNwYWNlOiBwcmUiIGZvbnQtZmFtaWx5PSJJQk0gUGxleCBTYW5zIiBmb250LXNpemU9IjEyIiBsZXR0ZXItc3BhY2luZz0iMGVtIj48dHNwYW4geD0iMTc1IiB5PSI1NzMuNSI+SW5jbHVkZWQgaW4gdGhpcyBhcmNoaXRlY3R1cmU8L3RzcGFuPjwvdGV4dD4KPC9nPgo8ZGVmcz4KPGNsaXBQYXRoIGlkPSJjbGlwMF8yMDM2XzIxNDE5NCI+CjxyZWN0IHdpZHRoPSIxMTQ1IiBoZWlnaHQ9IjU3NiIgZmlsbD0id2hpdGUiLz4KPC9jbGlwUGF0aD4KPC9kZWZzPgo8L3N2Zz4K',
```

Comment:
Open to suggestions of a better spot for this SVG. I tried an import, but there isn't a loader setup to handle SVG, so I thought this was an easy way to get an SVG here to test with.